boot/freeldr/freeldr/uefi.cmake

```diff
@@ -87,6 +91,7 @@ add_executable(uefildr ${UEFILDR_BASE_SOURCE})
 set_target_properties(uefildr PROPERTIES SUFFIX ".efi")
 
 target_compile_definitions(uefildr PRIVATE UEFIBOOT)
+set_image_base(uefildr 0x10000)
```

Comment:
Is this needed?

Reply:
so badly, UEFI threw freeloader into 8gb+ and I had to change loader pages span to adjust for 8gb!!! This is absolutely ridiculous.

and ReactOS explodes when there’s that many pages marked like this.

Reply:
That's interesting.... Worth its own investigation.

Reply:
Well UEFI is suppose to respect the base address of the PE file, and when the AMD64 linker doesn’t have one set it’ll assign it pretty high due to ASLR